packages/maskbook/src/extension/background-script/CryptoServices/encryptTo.ts

```diff
@@ -121,11 +121,11 @@ function getSummary(content: TypedMessageText) {
         // it seems like using "en" can also split the word correctly.
         const seg = new Intl.Segmenter('en')
         for (const word of seg.segment(content.content)) {
-            if (result.length >= 20) break
+            if (result.length >= 30) break
```

Comment:
The 30 looks like a magic number 👀, Why do you need to set it to 30

Reply:
The 20 chars is too short to show the twitter content summary.